README.md

```diff
@@ -1,2 +1,30 @@
 # install-scripts
 Installation scripts for OpenShift KNI clusters
+
+## KubeVirt
+The installation of KubeVirt related operators is managed by a 'meta operator'
+called the [HyperConverged Cluster Operator](https://github.com/kubevirt/hyperconverged-cluster-operator) (HCO).
+Deploying with the 'meta operator' will launch operators for KubeVirt,
+Containerized Data Imported (CDI), Cluster Network Addons (CNA),
+Common templates (SSP), and Node Maintenance Operator (NMO).
+
+#### Deploy the HCO through Marketplace
+The HyperConverged Cluster Operator is listed in the Community Marketplace,
+so you can go the rest of the way using the UI by clicking the 'OperatorHub' tab
+and selecting the HyperConverged Cluster Operator.
+
+If you want to use the CLI, run the following commands:
+```bash
+$ curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/master/deploy/hco.yaml | kubectl create -f -
```

Comment:
Will want to move to productized builds in time, and at that time would probably make sense to include this manifest here rather than reference something external that could change